### PR TITLE
Fixes translation layout

### PR DESCRIPTION
**Pull request by Fidel365**
Fixes (partially) #7598 
@N-thony and @derekagorhom this PR is ready for preview 
I fixed point 1 to point 4 of the issue

### CODE REVIEW

File: instat/dlgImportDataset.vb [hunk 1]
```diff
@@ -1141,6 +1141,10 @@ Public Class dlgImportDataset
         Return frmMain.clsRLink.MakeValidText(strCleanFileName)
     End Function
 
+    Private Sub lblImportingSheets_Click(sender As Object, e As EventArgs) Handles lblImportingSheets.Click
+
+    End Sub
```
Comment (N-thony) [inline instat/dlgImportDataset.vb:1146]: remove?

**Comment by Fidel365** [comment]
@N-thony I have made the changes requested

**Comment by N-thony** [comment]
> @N-thony I have made the changes requested

@Fidel365 we still the problem of controls overlapping on others in the French version
image

**Comment by N-thony** [comment]
> @N-thony and @derekagorhom I have made the changes requested except for number 5 because of the nature of the checkbox

@Fidel365 it looks fine when you try to import a `.csv file`, but not with a `.xlsx or .xls file`. You can try with any excel data file.
image

**Comment by Fidel365** [comment]
@N-thony  @derekagorhom , I have made the changes in excel too!

**Comment by Fidel365** [comment]
@N-thony, okay, am winding up on the other parts

**Comment by Fidel365** [comment]
@N-thony , I have fixed the remainder part of the issue!

**Comment by N-thony** [comment]
> @N-thony , I have fixed the remainder part of the issue!

@Fidel365 same time

1. can you make the first plot width same as the one just below?

image

2.  can you also make the change to this checkbox by increasing the width to 128?
image

3. also in the PICSA option subdialogue, can you increase the width of the checked checkboxes to fit the French?
image

**Comment by N-thony** [comment]
image

@Fidel365 I'm not able to test this, @rdstern @lloyddewit would you like to test? Thanks.

**Comment by Fidel365** [comment]
I'm having the same problem when I try to run it from my side, I think the
errors are coming from dlgImportDataset.Designer, I'll try to fix them

**Comment by Fidel365** [comment]
@N-thony , I'm done with number 1

**Comment by N-thony** [comment]
@Fidel365 is this ready for re-review?

**Comment by Fidel365** [comment]
@N-thony , it is ready

**Comment by Fidel365** [comment]
@N-thony , it's ready for the re-review

File: instat/dlgImportDataset.vb [hunk 2]
```diff
@@ -1141,6 +1141,10 @@ Public Class dlgImportDataset
         Return frmMain.clsRLink.MakeValidText(strCleanFileName)
     End Function
 
+    Private Sub grpCSV_Enter(sender As Object, e As EventArgs) Handles grpCSV.Enter
+
+    End Sub
```
Comment (N-thony) [inline instat/dlgImportDataset.vb:1146]: remove?

**Review by N-thony — CHANGES_REQUESTED** [review]

**Review by lloyddewit — DISMISSED** [review]

**Comment by lloyddewit** [comment]
@rdstern If you can test/approve, then we can merge, thanks

File: instat/dlgImportDataset.vb (hunk 2 above)
Comment (lloyddewit) [reply instat/dlgImportDataset.vb]: @N-thony this seems resolved now, so I'll go ahead and ask @rdstern to test

**Comment by N-thony** [comment]
> @rdstern If you can test/approve, then we can merge, thanks

@Fidel365 is working now on point 3.

**Review by N-thony — DISMISSED** [review]

**Review by lloyddewit — APPROVED** [review]

**Review by rdstern — APPROVED** [review]
I am approving, because it looks improved.
image

I note one item here not translated, but that may come elsewhere?

**Pull request merged**